pipeline/src/node_properties/track_properties.rs

```diff
@@ -31,10 +35,24 @@ pub enum TrackerProfile {
 
     /// If available for this tracker type, loads a configuration tuned for stationary objects
     Stationary,
+
+    /// Configuration tweaked by user
+    Custom,
```

Comment:
Not sure whether it's worth it but *I think* we could roll `TrackerCustomProperties` into this by using `serde(flatten)` on the `profile` field, `serde(tag = "profile", content = "custom_properties")` on `TrackerProfile`. Should add tests if going this route though.

Reply:
I will keep the current format for simplicity, but later we can revisit this if required.